metricbeat/helper/server/http/http_test.go

```diff
@@ -89,3 +192,23 @@ func writeToServer(t *testing.T, message, host string, port int) {
 	defer resp.Body.Close()
 
 }
+
+func writeToSslServer(t *testing.T, message, host string, port int) {
```

Comment:
what about adding the schema `{http|https}` as a parameter so we can have only `writeToServer` function, and depending on the schema we can build the URL and configure the client's transport.